internal/controller/deployment_to_pdb_controller.go

```diff
@@ -39,18 +40,29 @@ func (r *DeploymentToPDBReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	// Fetch the Deployment instance
 	var deployment v1.Deployment
 	if err := r.Get(ctx, req.NamespacedName, &deployment); err != nil {
-		return reconcile.Result{}, err
+		return reconcile.Result{}, client.IgnoreNotFound(err)
```

Comment:
nit should we be decrementing deployment gauge when not found? :). Don't have to decide now can create a todo or an issue to figure it out later.

Reply:
thats fair if its not foudn you don't know the label to decrement. fun. We could put a diffrent counter in here to see i this ever happens later fine for now.

Reply:
this kind of is the "seen" deployment gauge already?